sql_bench/README.md

```diff
@@ -0,0 +1,31 @@
+# Local e2e SQL Benchmarks
+
+This folder contains SQL benchmarks which are ran with [`hyperfine`](https://github.com/sharkdp/hyperfine).
+We can provide a template that wraps the `hyperfine` command,
+and runs it as a bash script.
+
+It's a little premature to design a full benchmark suite for now.
+
+You can provision a cloud instance and run the benchmarks there,
+or run them locally.
```

Comment:
BTW, we had https://github.com/risingwavelabs/risedev-bench  (unmaintained) for cloud bench